SetupDataPkg/Tools/VariableList.py

```diff
@@ -1240,14 +1245,15 @@ def write_csv(schema, csv_path, full, subknobs=True):
                         # We print the guid on the first row and then only print
                         # another guid if it changes
                         guid = subknob.namespace
-
+                        print("test1")

```

Comment:
can you remove the debug prints as well?

Reply:
Removed